### PR TITLE
Make showing User Guide link optional for jupyter widget

### PR DESCRIPTION
**Pull request by giswqs**
This pull request resolves the issue https://github.com/keplergl/kepler.gl/issues/1553. By default, the User Guide URL will always be printed when a map is created. Users can set `KeplerGl(show_docs=False)` to hide the User Guide URL if needed. 

Before:

After:

### CODE REVIEW

**Review by heshan0131 — APPROVED** [review]

**Comment by heshan0131** [comment]
https://github.com/keplergl/kepler.gl/blob/master/docs/keplergl-jupyter/README.md

**Comment by giswqs** [comment]
@heshan0131 I have added the following to the documentation. 


By default, the User Guide URL (<https://docs.kepler.gl/docs/keplergl-jupyter>) will be printed when a map is created. To hide the User Guide URL, set `show_docs=False`.

```python
# Hide the User Guide URL
from keplergl import KeplerGl
map_1 = KeplerGl(show_docs=False)
map_1
```

File: docs/keplergl-jupyter/README.md [hunk 1]
```diff
@@ -90,6 +90,16 @@ map_1
 
 ![empty map][empty_map]
 
+ By default, the User Guide URL (<https://docs.kepler.gl/docs/keplergl-jupyter>) will be printed when a map is created. To hide the User Guide URL, set `show_docs=False`.
```
Comment (heshan0131) [inline docs/keplergl-jupyter/README.md:93]: Move this to after `__`config`__` on `line 77`

Comment (giswqs) [reply docs/keplergl-jupyter/README.md]: Updated. Thanks. 

image

**Comment by heshan0131** [comment]
target release `keplergl 0.3.1`

**Pull request merged**